app/src/main/java/io/zenandroid/onlinego/ui/composables/BoardComposable.kt

```diff
@@ -58,13 +62,30 @@ fun Board(
     onTapUp: ((Cell) -> Unit)? = null
 ) {
     BoxWithConstraints(modifier = modifier.aspectRatio(1f)) {
+        val settingsRepository: SettingsRepository = GlobalContext.get().get()
```

Comment:
I had trouble because the MyGameUI would not update the boards when the setting changed. We had no problem with coordinates as they are never shown here whatever the setting is.
You'll see that I made a change in the onViewResumed() of MyGamesViewModel, there is certainly a much more beautiful way to achieve this, but I'm not confident with the update lifecycle of this part...